.github/workflows/test-application.yaml

```diff
@@ -16,13 +16,7 @@ jobs:
             fail-fast: false
             matrix:
                 include:
-                    - php-version: '5.6'
-                      elasticsearch-version: '5.6.14'
-                      lint: false
-                      symfony-version: '^2.8'
-                      elasticsearch-package-constraint: '^5.0'
-
-                    - php-version: '7.0'
+                    - php-version: '7.2'
```

Comment:
the new version of the `symfony/phpunit-bridge` does not include the `SetUpTearDownTrait` anymore. therefore we cannot support php versions that are compatible with php@5.6

Reply:
we should drop then php@5.6 from the composer.json

Reply:
done 🙂